src/DynamoCoreWpf/ViewModels/Core/ConnectorPinViewModel.cs

```diff
@@ -119,10 +119,10 @@ public double Left
         /// </summary>
         public double Top
         {
-            get { return model.Y; }
+            get { return model.Y- OneThirdWidth; }
```

Comment:
can you add comments why 1/3 but not 1/2?

Reply:
hmm honestly the 1/3 value resulted from trial and error. can add that as a comment.